### PR TITLE
Add deprecation/removal notes to the 0.1.x migration service.

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: docs/synapse/devguides/devops_cortex.rst [hunk 1]
```diff
@@ -89,6 +89,10 @@ the Axon.
 0.1.x to 2.x.x Migration
 ------------------------
 
+.. warning::
+    The 0.1.x to 2.x.x migration tools will be removed in Synapse v2.9.0. The tools
+    will continue to be available in previous version of the 2.x.x releases.
```
Comment (invisig0th) [inline docs/synapse/devguides/devops_cortex.rst:94]: Maybe a quick line about the upgrade path being 0.1.x -> 2.8.X -> fully updated?

Comment (vEpiphyte) [reply docs/synapse/devguides/devops_cortex.rst]: @invisig0th I refined this a bit

Comment (vEpiphyte) [reply docs/synapse/devguides/devops_cortex.rst]: Test

File: docs/synapse/devguides/devops_cortex.rst [hunk 2]
```diff
@@ -89,6 +89,11 @@ the Axon.
 0.1.x to 2.x.x Migration
 ------------------------
 
+.. warning::
+    The ``0.1.x`` to ``2.x.x`` migration tools will be removed in Synapse ``v2.9.0``. The
+    tools will continue to be available in previous version of the ``2.x.x`` releases.
```
Comment (mikemoritz) [inline docs/synapse/devguides/devops_cortex.rst:94]: versions

File: synapse/tools/migrate_200.py [hunk 3]
```diff
@@ -618,6 +618,8 @@ class Migrator(s_base.Base):
     '''
     async def __anit__(self, conf):
         await s_base.Base.__anit__(self)
+
+        logger.warning('The 0.1.x to 2.0.0 migration code will be removed from Synapse in 2.9.0')
```
Comment (mikemoritz) [inline synapse/tools/migrate_200.py:622]: migration tool instead of code?

**Review by mikemoritz — APPROVED** [review]

**Pull request merged**